### ISSUE TITLE
A11y_Fluent UI_Components_Fluent UI React v9_AI4W: Color contrast ratio of the web react v9 link is less than 3:1

### ISSUE THREAD

**Issue by Latha2564** [microsoft/vscode-azurefunctions#3402]
**GitHubTags:**

#A11ySev2; #TCS-E+d; #NFT-Fluentui-Web-Oct2022; #DesktopWeb; #WCAG1.4.3;
 #A11yMAS;

**OS Details:**

Web: win11

**Environment Details:**

Application Name: Fluent Ui

**Repro Steps:**

1.Hit enter on the URL https://react-dev.fluentui.dev/
2."Fluent UI React v9" page will open.
3.Tab till "Web React v9" link on the left side bar and hit enter. 
4."Fluent Web React v9" page will open.
5.Verify Color contrast ratio of the web react v9 link is less than 3:1

**Actual Result**

Color contrast ratio of the web react v9 link is less than 3:1

**Expected Result:**

Color contrast ratio of the web react v9 link should be greater than or equal to 3:1

**User Impact:**

Color contrast user will face difficulties when color contrast ratio of the web react v9 link should be greater than or equal to 3:1

**Comment by Latha2564** [comment]
closing this bug as the bug logged in wrong ADO
